test/functional/transactions/0.tx.js

```diff
@@ -0,0 +1,637 @@
+'use strict';
+
+var _ = require('lodash');
+var crypto = require('crypto');
+
+var node = require('../../node.js');
+var http = require('../../common/httpCommunication.js');
+var sendTransaction = require('../../common/complexTransactions.js').sendTransaction;
+
+function getTransaction (transaction, cb) {
+	http.get('/api/transactions/get?id='+transaction, function (err, res) {
+		if (err) {
+			return cb(err);
+		}
+		node.expect(res.body).to.have.property('success');
+		cb(null, res.body);
+	});
+}
+
+describe('postTransactions type 0', function () {
+
+	var badTransactions = [];
+	var goodTransactions = [];
+
+	describe('schema', function () {
+
+		var transaction;
+
+		beforeEach(function () {
+			transaction = node.randomTx();
+		});
+
+		it('using null transaction should fail', function (done) {
+			sendTransaction(null, function (err, res) {
+				node.expect(res).to.have.property('success').to.not.be.ok;
+				node.expect(res).to.have.property('message').to.equal('Invalid transaction body - Empty trs passed');
+				done();
+			}, true);
+		});
+
+		it('using empty transaction should fail', function (done) {
+			sendTransaction({}, function (err, res) {
+				node.expect(res).to.have.property('success').not.to.be.ok;
+				node.expect(res).to.have.property('message').to.equal('Invalid transaction body - Empty trs passed');
+				done();
+			}, true);
+		});
+
+		it('using undefined transaction should fail', function (done) {
+			sendTransaction(null, function (err, res) {
+				node.expect(res).to.have.property('success').to.not.be.ok;
+				node.expect(res).to.have.property('message').to.equal('Invalid transaction body - Empty trs passed');
+				done();
+			}, true);
+		});
+
+		it('using empty array should fail', function (done) {
+			sendTransaction([], function (err, res) {
+				node.expect(res).to.have.property('success').to.not.be.ok;
+				node.expect(res).to.have.property('message').to.equal('Invalid transaction body - Empty trs passed');
+				done();
+			}, true);
+		});
+
+		it('using NOT defined type should fail', function (done) {
+			transaction.type = 100;
+
+			sendTransaction(transaction, function (err, res) {
+				node.expect(res).to.have.property('success').to.not.be.ok;
+				node.expect(res).to.have.property('message').to.match(/^Invalid transaction body - Unknown transaction type/);
```

Comment:
Depending on the transaction type that you send, it receives a customised message as for example `Invalid transaction body - Unknown transaction type string`or `Invalid transaction body - Unknown transaction type xxx`.